packages/dynamodb-data-mapper/src/DataMapper.ts

```diff
@@ -300,6 +301,13 @@ export class DataMapper {
             StreamSpecification: streamViewType === 'NONE'
                 ? { StreamEnabled: false }
                 : { StreamEnabled: true, StreamViewType: streamViewType },
+            SSESpecification: sseSpecification.enabled
```

Comment:
Is there a use case where someone might change their SSE settings but not enable it? Could this instead check if `sseSpecification` is defined?